Casks/chessx.rb

```diff
@@ -9,8 +9,8 @@
   homepage "https://chessx.sourceforge.io/"
 
   livecheck do
-    url "https://sourceforge.net/projects/chessx/files/latest/download"
-    strategy :header_match
+    url "https://sourceforge.net/projects/chessx/rss?path=/chessx"
```

Comment:
```suggestion
    url "https://sourceforge.net/projects/chessx/rss
```
In addition to the great suggestions below, I think the url can also be simplified. 

Not an issue in this specific cask, but may be worth highlighting some of the potential considerations when using the implied [Sourceforge](https://rubydoc.brew.sh/Homebrew/Livecheck/Strategy/Sourceforge.html) strategy vs PageMatch.

Reply:
Thank you @razvanazamfirei!  Valid point.  Should've seen that.  😄

Made the change

Reply:
@razvanazamfirei In this case, the original URL (with the `path` parameter) was preferable here. This restricts the RSS feed to only the `chessx` directory, which is where the release files are kept.

Basically, the idea is that we do this when appropriate/possible because the RSS feed contains a limited number of entries. The URL without the path will also contain entries from the `chessx-extras` directory and, theoretically, the files we're interested in could be pushed out of the feed by files we're not interested in.

Hope this helps to explain this URL pattern across other casks but let me know if you have questions.

Reply:
@samford, thank you for making the clarification regarding the url. On first glance, the files looked identical and my local tests gave the same results, which is why I suggested the simpler version. 

>It may also be worth mentioning that this livecheck block still uses the Sourceforge strategy with these changes (not PageMatch directly)

Yeah, this is exactly what I was trying to point out; thank you for stating it much more clearly than I could and for all your detailed explanations. I'm always low-key excited reading your comments.

Reply:
Thanks @samford and @razvanazamfirei for your time and explanation of the nuances for how this works. This level of education is superb.

I understand your primary goal is stability and making correct changes, and as someone new to submitting PR's for the project I appreciate the specificity, kindness, and willingness to work to make everything correct.

Thank you!